filter-effects-2/Overview.bs

```diff
@@ -50,28 +54,281 @@ If the value of the 'backdrop-filter' property is ''backdrop-filter/none'' then
+
+Note that the effect of the backdrop-filter will not be visible unless some
+portion of element B is semi-transparent. Also note that any opacity applied to
+element B will be applied to the filtered backdrop image as well. Therefore, to
+create a "transparent" element that allows the full filtered backdrop image to
+be seen, you can use "background-color: transparent;".
+
+<pre class=link-defaults>
+  spec:filter-effects-1; type:element; text:filter
+</pre>
+
+The first <a element>filter</a> function or filter reference in the <<filter-value-list>>
+takes the element's <a>Backdrop Root Image</a> as the input image. Subsequent
 operations take the output from the previous filter function or <a
-element>filter</a> reference as the input image. <a element>filter</a> element
+element>filter</a> reference as the input image. The <a element>filter</a> element
 reference functions can specify an alternate input, but still uses the previous
 output as its SourceGraphic.
 
 Filter functions must operate in the sRGB color space.
 
-A computed value of other than ''backdrop-filter/none'' results in the creation of a <a
-href="https://www.w3.org/TR/CSS21/zindex.html">stacking context</a> [[!CSS21]]
-the same way that CSS 'opacity' does. All the elements descendants are rendered
-together as a group with the filter effect applied to the group as a whole.
+Note: A computed value of other than ''backdrop-filter/none'' results in the
+creation of both a <a href="https://www.w3.org/TR/CSS21/zindex.html">stacking
+context</a> [[!CSS21]] and a <a
+href="https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block">Containing
+Block</a> for absolute and fixed position descendants, in the same way that CSS
+'filter' does.
+
+# Backdrop Root # {#BackdropRoot}
+
+The <dfn>Backdrop Root Image</dfn> is defined as the entire painted output produced by
+drawing, in <a
+href="https://www.w3.org/TR/CSS2/zindex.html#painting-order">painting order</a>,
+all content between (and including) the <a>Backdrop Root</a> element and the
+child element that needs access to the "backdrop". No content that is a DOM
+ancestor of the Backdrop Root element should contribute to or affect the
+Backdrop Root Image.
+
+A <dfn>Backdrop Root</dfn> is formed, anywhere in the document, by an element in any of the following scenarios. See [[#BackdropRootTriggers]] for more details on each:
+ * The root element of the document (HTML).
+ * An element with a filter property other than "none".
+ * An element with an opacity value less than 1.
+ * An element with mask, mask-image, mask-border, or clip-path properties with values other than “none”.
+ * An element with a backdrop-filter value other than "none".
+ * An element with a mix-blend-mode value other than "normal".
+ * An element with a will-change value specifying any property that would create a Backdrop Root on non-initial value.
+
+Note: this definition encompases fewer element types than the definition for a
```

Comment:
Suggest removing this paragraph.

Reply:
See comment further up from mstange@ - he suggested moving this paragraph to here. I do like it, somewhere in the document. Thoughts?

Reply:
ok